readme.txt

```diff
@@ -2,17 +2,20 @@
 Contributors: comfino.pl
 Donate link: https://comfino.pl/
 Tags: comfino, woocommerce, gateway, payment, bank, deferred payments, installments
-WC tested up to: 8.1.1
+WC tested up to: 9.0.2
 WC requires at least: 3.0
-Stable tag: 3.4.1
+Stable tag: 3.4.2
 Tested up to: 6.3.1
 Requires at least: 5.0
 Requires PHP: 7.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
+3.4.2
+ Mark plugin as compatible with WooCommerce HPOS function.
```

Comment:
Jakby było README w formacie `md` można by było link wstawić, bo się zastanwiałem co to jest.

```suggestion
 Mark plugin as compatible with WooCommerce [HPOS](https://woocommerce.com/document/high-performance-order-storage/) function.
```

Reply:
W sumie w nawias można wrzucić też w txt. Dodam link. Ja do czasu tej poprawki też nie wiedziałem. ;-)